system/include/emscripten/em_asm.h

```diff
@@ -0,0 +1,32 @@
+#ifndef __em_asm_h__
+#define __em_asm_h__
+
+#ifndef __cplusplus
+void emscripten_asm_const();
```

Comment:
Should probably have a comment that explicitly calls out that these are non-prototype declarations whose purpose is to be valid ISO C (which requires declarations) while still allowing non-vararg calls with different parameters.